### PR TITLE
Update nixpkgs unstable

### PR DESCRIPTION
**Pull request by airportyh**
Why
===

Update nixpkgs-stable to get [glibc security vulnerability fix](https://github.com/NixOS/nixpkgs/pull/285050)

What changed
============

Updated flake input for nixpkgs-unstable

Test plan
=========

1. run template tests

### See that glibc update is there:

1. create new python repl
2. In the shell
```
python
> import sys
sys.executable
# will show full path to Python executable (not the wrapper)
```
3. copy the path, we'll call that `PYTHON_EXE_PATH`
4. readelf -d `PYTHON_EXE_PATH`
5. Find `glibc-2.38-44` in the `(RUNPATH)` section of the output, (not `glibc-2.38-27`)

### CODE REVIEW

**Comment by airportyh** [comment]
looks like the build failed because it's making requests to the cache too frequently :/ 
image

**Comment by airportyh** [comment]
Wait, that might be an outgoing rate limit from Github...

**Comment by airportyh** [comment]
Let me try running the ci check manually...

**Comment by airportyh** [comment]
:(
```
python3.10-replit-python-dap-wrapper> !!
python3.10-replit-python-dap-wrapper>         ********************************************************************************
python3.10-replit-python-dap-wrapper>         Requirements should be satisfied by a PEP 517 installer.
python3.10-replit-python-dap-wrapper>         If you are using pip, you can try `pip install --use-pep517`.
python3.10-replit-python-dap-wrapper>         ********************************************************************************
python3.10-replit-python-dap-wrapper> !!
python3.10-replit-python-dap-wrapper>   tr_d = dist.fetch_build_eggs(dist.tests_require or [])
python3.10-replit-python-dap-wrapper> /nix/store/jj0ngw5gls74iqsh9hq7db769g0zwsxj-python3.10-setuptools-69.0.3/lib/python3.10/site-packages/setuptools/command/test.py:195: _DeprecatedInstaller: setuptools.installer and fetch_build_eggs are deprecated.
python3.10-replit-python-dap-wrapper> !!
python3.10-replit-python-dap-wrapper>         ********************************************************************************
python3.10-replit-python-dap-wrapper>         Requirements should be satisfied by a PEP 517 installer.
python3.10-replit-python-dap-wrapper>         If you are using pip, you can try `pip install --use-pep517`.
python3.10-replit-python-dap-wrapper>         ********************************************************************************
python3.10-replit-python-dap-wrapper> !!
python3.10-replit-python-dap-wrapper>   er_d = dist.fetch_build_eggs(
python3.10-replit-python-dap-wrapper> running egg_info
python3.10-replit-python-dap-wrapper> writing replit_python_dap_wrapper.egg-info/PKG-INFO
python3.10-replit-python-dap-wrapper> writing dependency_links to replit_python_dap_wrapper.egg-info/dependency_links.txt
python3.10-replit-python-dap-wrapper> writing top-level names to replit_python_dap_wrapper.egg-info/top_level.txt
python3.10-replit-python-dap-wrapper> reading manifest file 'replit_python_dap_wrapper.egg-info/SOURCES.txt'
python3.10-replit-python-dap-wrapper> writing manifest file 'replit_python_dap_wrapper.egg-info/SOURCES.txt'
python3.10-replit-python-dap-wrapper> running build_ext
python3.10-replit-python-dap-wrapper> ----------------------------------------------------------------------
python3.10-replit-python-dap-wrapper> Ran 0 tests in 0.000s
python3.10-replit-python-dap-wrapper> OK
python3.10-replit-python-dap-wrapper> Finished executing setuptoolsCheckPhase
Traceback (most recent call last):
  File "/app/nixmodules/scripts/build_changed_modules.py", line 60, in <module>
    main()
  File "/app/nixmodules/scripts/build_changed_modules.py", line 55, in main
    assert actual_path == referenced_path, 'output path for %s does not match: %s vs %s' % (module_registry_id, actual_path, referenced_path)
AssertionError: output path for python-3.10:v53-20240306-f3ca166 does not match: /nix/store/g4n84pj5wdq89p2mhmp314f7a57waxi5-replit-module-python-3.10 vs /nix/store/7s4d5lzgs844czgn0sjl76sc7lzqqbyk-replit-module-python-3.10
```

**Review by blast-hardcheese — APPROVED** [review]

File: scripts/show_reg.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      'name': packager['name'],
+      'displayVersion': packager['displayVersion'],
+      'moduleId': packager['moduleId']
+    })
+print('Formatters:')
+print(json.dumps(formatters, indent='  '))
+print('Language Servers:')
+print(json.dumps(languageServers, indent='  '))
+print('Packagers:')
+print(json.dumps(packagers, indent='  '))
```
Comment (blast-hardcheese) [inline scripts/show_reg.py:37]:
```suggestion
print(json.dumps(packagers, indent='  '))

```

**Review by blast-hardcheese — APPROVED** [review]
one question

**Comment by airportyh** [comment]
Switching to https://github.com/replit/nixmodules/pull/286

**Pull request closed without merging**